packages/inventory/src/api/index.js

```diff
@@ -24,12 +24,28 @@ export const mapData = ({ facts = {}, ...oneResult }) => ({
         .reduce(
             (acc, curr) => ({ ...acc, ...(typeof curr !== 'string') ? curr : {} }), {}
         )
-    },
-    ...localStorage.getItem('rhcs-tags') === 'true' ? {
-        tagCount: Math.floor(Math.random() * Math.floor(11))
-    } : {}
+    }
 });
 
+export const mapTags = (data = { results: [] }, { orderBy, orderDirection } = {}) => {
+    if (data.results.length > 0) {
+        return hosts.apiHostGetHostTags(data.results.map(({ id }) => id), data.per_page, 1, orderBy, orderDirection)
+            .then(({ results: tags }) => ({
+                ...data,
+                results: data.results.map(row => ({ ...row, tags: tags[row.id] || [] }))
+            }))
+            .catch(() => ({
```

Comment:
This is safe way how not to fail on fetching tags (they are something trivial and shouldn't break the UI)